src/main/java/ch/njol/skript/expressions/ExprJoinSplit.java

```diff
@@ -42,15 +42,23 @@
 @Description("Joins several texts with a common delimiter (e.g. \", \"), or splits a text into multiple texts at a given delimiter.")
 @Examples({"message \"Online players: %join all players with \"\" | \"\"%\" # %all players% would use the default \"x, y, and z\"",
 		"set {_s::*} to the string argument split at \",\""})
-@Since("2.1")
+
```

Comment:
Im not sure if I didn't notice this before, or its new, but there shouldn't be a space here